.github/workflows/ci.yml

```diff
@@ -80,6 +80,10 @@ jobs:
             modin/engines/base/io/__init__.py \
             modin/engines/base/io/file_dispatcher.py
       - run: python scripts/doc_checker.py modin/utils.py
+      - run: |
+          python scripts/doc_checker.py modin/experimental/xgboost/__init__.py \
+            modin/experimental/xgboost/utils.py modin/experimental/xgboost/xgboost.py \
+            modin/experimental/xgboost/xgboost_ray.py
```

Comment:
```suggestion
          python scripts/doc_checker.py modin/experimental/xgboost
```

Reply:
Directory `modin/experimental/xgboost` also contains `test` directory, which isn't documented. Can we exclude `scripts/doc_checker.py modin/experimental/xgboost/test/` from testing in `doc_checker.py`? I am not sure we need add docs to all tests.

Reply:
I don't think we can exclude, so let's leave this as it is now, thanks for explanation.

Reply:
@vnlitvinov I tried to add `python scripts/doc_checker.py modin/experimental/xgboost` instead of sequence of files in `ci`. `pydocstyle check` passed, because the lack of docs in files doesn't  throw "failed" status for the check. I use your suggestion with directory. What do you think?